src/server/scripts/Northrend/Naxxramas/boss_four_horsemen.cpp

```diff
@@ -182,7 +182,7 @@ class boss_four_horsemen : public CreatureScript
             events.Reset();
             // Schedule Events
             events.RescheduleEvent(EVENT_SPELL_MARK_CAST, 24000);
-            events.RescheduleEvent(EVENT_BERSERK, 100 * 15000);
+            events.RescheduleEvent(EVENT_BERSERK, 600000);
```

Comment:
what about using constants like in other places? such as `5 * MINUTES` ? so it's more readable

Reply:
we work in milliseconds, check 99% of scripts.